File: js/romanNumerals.js
exports.toRoman = function(num) {
    


    const numerals = ['M','CM','D','CD','C','XC','L','XL','X','IX','V','IV','I'];
    const integers = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1];
    let result = '';

    while (num !== 0) {
        for (let i = 0; i < numerals.length; i++) {
            if (num >= integers[i]) {
                result += numerals[i];
                num -= integers[i];
                break;
            }
        }
    }
    return result;
};

File: python/roman_numerals.py
def to_roman(num):
    roman_nums = {
        1000: "M",
        900: "CM",
        500: "D",
        400: "CD",
        100: "C",
        90: "XC",
        50: "L",
        40: "XL",
        10: "X",
        9: "IX",
        5: "V",
        4: "IV",
        1: "I",
    }

    res = ""

    for i in roman_nums:
        res += roman_nums[i] * (num // i)
        num %= i

    return res







print(to_roman(1994))
